File: src/main/java/com/monkeyteam/monkeyutils/modules/MOTD.java
package com.monkeyteam.monkeyutils.modules;

import com.monkeyteam.monkeyutils.MonkeyUtils;
import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;

import java.util.List;

public class MOTD implements Listener {

    private final List<String> messages;

    public MOTD() {
        messages = MonkeyUtils.getInstance().getConfig().getStringList("motd");
    }

    @EventHandler
    public void onPlayerJoin(PlayerJoinEvent event) {
        Player player = event.getPlayer();
        messages.forEach(player::sendMessage);
    }

}

File: src/main/java/com/monkeyteam/monkeyutils/modules/MOTD.kt
package com.monkeyteam.monkeyutils.modules

import com.monkeyteam.monkeyutils.MonkeyUtils
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent

object MOTD : Listener {
  private val messages: List<String> = MonkeyUtils.instance.config.getStringList("motd")

  @EventHandler
  fun onPlayerJoin(event: PlayerJoinEvent) {
    val player = event.player
    messages.forEach(player::sendMessage)
  }
}
